### PR TITLE
Issue-553 Documentation for new API (WIP)

### PR DESCRIPTION
**Pull request by eolivelli**
Add documentation for new API

### CODE REVIEW

**Comment by eolivelli** [comment]
@sijie can you take a first look ?

**Comment by jiazhai** [comment]
Seems #727 is going on, should we merge this after that, and change some of the description according to it?

**Comment by asfgit** [comment]
FAILURE
 
--none--

**Comment by sijie** [comment]
The API documentation should be the final change after we sort out all the api related issues.

File: site/docs/4.6.0/api/ledger-api.md [hunk 1]
```diff
@@ -264,6 +264,88 @@ Result: 2
 # etc
 ```
 
+## New API
+
+Since 4.6 BookKeeper provides a new client API which leverages Java CompletableFuture facility.
```
Comment (sijie) [inline site/docs/4.6.0/api/ledger-api.md]: you need also document 

- delete op
- append api in WriteHandle
- read api in ReadHandle

Also you need to update ledgeradv api page as well.

**Comment by sijie** [comment]
@eolivelli can you help pick this up? This will be blocking 4.6.0.

**Comment by eolivelli** [comment]
Yes. I stopped for a while. I will address comments today

**Comment by sijie** [comment]
@eolivelli this issue has higher priority than BP18. would appreciate you help completed this. if you don't have time, I am okay to take it.

**Comment by eolivelli** [comment]
I am doing this today (for me is early morning now), I will update the pr soon. I know this is a priority.
On 4.6 we have some other blocker, like the LedgerEntries stuff.

I think you are faster for many reasons.
You can pick this up and finish the work.
I am sorry

**Comment by sijie** [comment]
@eolivelli it is not clear to me. are you picking this up, or not? 

regarding other blockers, we are trying to clean them up this week.

**Comment by eolivelli** [comment]
Please pick this up, I think it is better.

**Comment by sijie** [comment]
@eolivelli I sent the pull request #764 . I will close this one since there is no activity on this one anymore.

**Pull request closed without merging**